### PR TITLE
Proposing Update to index.md - Env Settings for Rancher Desktop

### PR DESCRIPTION
**Pull request by sunilarjun**
Hello there! I am a technical writer for Rancher Desktop, and am proposing an update to index.md with environment settings for macOS M1 Rancher Desktop installations that I think would be helpful to testcontainers users. Please let me know if you need any more information, thank you.

### CODE REVIEW

File: docs/supported_docker_environment/index.md [hunk 1]
```diff
@@ -60,6 +60,34 @@ export TESTCONTAINERS_RYUK_DISABLED=true
...
+## Rancher Desktop
+
+In order to run testcontainers against [Rancher Desktop](https://rancherdesktop.io/) the env vars below should be set
+
+If you're running Rancher Desktop as an administrator in a MacOS (M1) machine:
+
+Using QEMU emulation
+
+```bash
+export TESTCONTAINERS_HOST_OVERRIDE=$(rdctl shell ip a show rd0 | awk '/inet / {sub("/.*",""); print $2}')
```
Comment (eddumelendez) [inline docs/supported_docker_environment/index.md:72]: on M1 with QEMU selected, I'm not able to get this value

**Review by eddumelendez — COMMENTED** [review]
Thank you so much for your contribution, @sunilarjun ! Very glad to see this contribution. I've left one question.

File: docs/supported_docker_environment/index.md (hunk 1 above)
Comment (sunilarjun) [reply docs/supported_docker_environment/index.md:72]: Hello, thank you for the review! With QEMU there is an additional requirement where you must allow admin access in Rancher Desktop (which can be set via the [Preferences > Application > General](https://docs.rancherdesktop.io/ui/preferences/application/general) dialog) and then the setting can be exported. If you wish to not allow admin access, then the VZ emulation must be used, along with the VZ export settings. I hope this clarifies and please let me know if there are any more questions.

Comment (eddumelendez) [reply docs/supported_docker_environment/index.md:72]: Thanks! I was missing that step.

**Review by eddumelendez — DISMISSED** [review]
LGTM

File: docs/supported_docker_environment/index.md (hunk 1 above)
Comment (sunilarjun) [reply docs/supported_docker_environment/index.md:72]: No worries!

File: docs/supported_docker_environment/index.md [hunk 2]
```diff
@@ -60,6 +60,34 @@ export TESTCONTAINERS_RYUK_DISABLED=true
     Previous to version 1.19.0, `export TESTCONTAINERS_RYUK_PRIVILEGED=true`
     was required for rootful mode. Starting with 1.19.0, this is no longer required.
 
+## Rancher Desktop
+
+In order to run testcontainers against [Rancher Desktop](https://rancherdesktop.io/) the env vars below should be set
```
Comment (kiview) [inline docs/supported_docker_environment/index.md:65]:
```suggestion
In order to run testcontainers against [Rancher Desktop](https://rancherdesktop.io/) the env vars below should be set.
```

**Review by kiview — APPROVED** [review]
Can we also add instructions for Windows?

**Comment by eddumelendez** [comment]
Thanks for your contribution, @sunilarjun! This section is now live in our docs too. See https://java.testcontainers.org/supported_docker_environment/#rancher-desktop

**Pull request merged**